frontend/src/components/Start.js

```diff
@@ -0,0 +1,12 @@
+// import React from 'react';
+// import { Link } from 'react-router-dom';
+
+
+
+// export const Start = () => {
+//     return(
+//         <>
+//     <Link to ='/registration'> Are You Ready to Talk?</Link>
+//         </>
+//     )
+// }
```

Comment:
why did you comment-out a whole component? You just need to unmount it from the other components.

Reply:
Just wanted to make it easy to see (for ourself) that we did not use it.